incubator/binding-mqtt-kafka/src/test/java/io/aklivity/zilla/runtime/binding/mqtt/kafka/internal/stream/MqttKafkaSessionProxyIT.java

```diff
@@ -292,28 +373,28 @@ public void shouldReconnectWillStreamOnKafkaReset() throws Exception
         k3po.finish();
     }
 
-    public static String supplySessionId()
+    public static MqttKafkaConfiguration.StringSupplier supplySessionId()
     {
-        return "sender-1";
+        return () -> "sender-1";
     }
 
-    public static String supplyWillId()
+    public static MqttKafkaConfiguration.StringSupplier supplyWillId()
     {
-        return "abcdabcddcbadcba";
+        return () -> "d252a6bd-abb5-446a-b0f7-d0a3d8c012e2";
     }
 
-    public static String supplyLifetimeId()
+    public static MqttKafkaConfiguration.StringSupplier supplyLifetimeId()
     {
-        return "bbbbbbbbbbbbbbbb";
+        return () -> "1e6a1eb5-810a-459d-a12c-a6fa08f228d1";
     }
 
-    public static String supplyInstanceId()
+    public static MqttKafkaConfiguration.StringSupplier supplyInstanceId()
     {
-        return "zilla-1";
+        return () -> "zilla-1";
     }
 
-    public static long supplyTime()
+    public static LongSupplier supplyTime()
     {
-        return 1000L;
+        return () -> 1000L;
```

Comment:
These static method changes can be reverted after applying `MqttKafkaConfiguration` feedback.